hub/core/compression.py

```diff
@@ -5,7 +5,12 @@
     UnsupportedCompressionError,
     CorruptedSampleError,
 )
-from hub.compression import get_compression_type
+from hub.compression import (
+    get_compression_type,
+    BYTE_COMPRESSION,
+    IMAGE_COMPRESSION,
+    AUDIO_COMPRESSION,
+)
 from typing import Union, Tuple, Sequence, List, Optional, BinaryIO
 import numpy as np
```

Comment:
The import aren't sorted here. Maybe you can use tools like black and isort to format the code so that the resulting code is aligned with the PEP 8 guidelines.